src/main/io/osd.c

```diff
@@ -1236,6 +1313,26 @@ static void osdDrawElements(void)
         osdDrawSingleElement(OSD_LOG_STATUS);
     }
 #endif
+
+#ifdef USE_OSD_STICK_OVERLAY
+#ifdef USE_OSD_PROFILES
```

Comment:
This code construction introduces double opening `{`, confusing some editors (yes, some of us are using emacs ;-) 

IMO `USE_OSD_PROFILES` should override `VISIBLE` macro(or use some other abstraction), so that OSD profiles are transparent to this type of code

Reply:
Oh yes, I like that idea.  Done.